[
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "Outer exception",
          "type": "System.Exception",
          "stacktrace": "Unknown.lambda_method<scrubbed>"
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "Outer exception"
              },
              "InnerException": {
                "type": "Exception",
                "value": "Exception",
                "fields": {
                  "Message": {
                    "type": "String",
                    "value": "Inner exception"
                  },
                  "InnerException": {
                    "type": "Exception",
                    "isNull": "true"
                  },
                  "HelpLink": {
                    "type": "String",
                    "isNull": "true"
                  },
                  "Source": {
                    "type": "String",
                    "value": "app"
                  },
                  "HResult": {
                    "type": "Int32",
                    "value": "-2146233088"
                  },
                  "StackTrace": {
                    "type": "String",
                    "value": "<scrubbed>"
                  }
                }
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          },
          "ex": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "Inner exception"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "ExceptionReplayInner",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": "Unknown.lambda_method<scrubbed>",
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  }
]